### DISCUSSION TITLE
Failed to parse source map: webpack URL is not supported

### DISCUSSION THREAD

**Discussion by Subhrangsu90** [webpack/webpack#17178] [Q&A] [answered]
`WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
![Screenshot (10)](https://github.com/webpack/webpack/assets/76052002/069b9165-2961-4c00-9b8e-e20a4fc9f0b2)
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/address.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/anonymous.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/array.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/boolean.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/bytes.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/fixed-bytes.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/null.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/number.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/string.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/coders/tuple.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/ens-normalize/decoder.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/ens-normalize/include.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src.ts/ens-normalize/lib.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src/chains/index.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src/eips/index.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../../src/hardforks/index.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/_version.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/abi-coder.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/addresses.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/bignumber.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/bignumbers.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/browser-base64.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/bytes32.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/fixednumber.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/fragments.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/hashes.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/id.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/idna.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/index.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/interface.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/message.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/namehash.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/strings.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/typed-data.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src.ts/utf8.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/account.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/address.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/baseTransaction.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/bytes.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/constants.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/eip1559Transaction.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/eip2930Transaction.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/externals.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/hash.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/helpers.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/index.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/internal.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/legacyTransaction.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/object.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/signature.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/transactionFactory.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/types.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///../src/util.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///src/hash-utils.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///src/keccak.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///src/random.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

WARNING in ./node_modules/web3/dist/web3.min.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map: 'webpack:///src/secp256k1.ts' URL is not supported
 @ ./src/components/App.js 13:0-24 71:32-36 79:34-38
 @ ./src/index.js 7:0-35 10:38-41

58 warnings have detailed information that is not shown.
Use 'stats.errorDetails: true' resp. '--stats-error-details' to show it.

webpack 5.69.1 compiled with 58 warnings in 1746 ms`

**Comment by alexander-akait** [comment]
`source-map-loader` is not supported `webpack://` protocol, please open an issue in `source-map-loader` repo and provide example of code how are you using

**Reply by Subhrangsu90** [reply, marked as answer]
Thank you 

**I solve this problem adding this configuration**  
**webpack.config.js**

```js
module.exports = {
  module: {
    rules: [
      {
        test: /\.js$/,
        enforce: "pre",
        use: ["source-map-loader"],
      },
    ],
  },
  ignoreWarnings: [/Failed to parse source map/],
};
```

**Reply by An-anson** [reply]
where can I find webpack.config.js?
